configs/projects/openvox-agent.rb

```diff
@@ -25,9 +25,8 @@
     proj.extra_file_to_sign File.join(proj.bindir, 'puppet')
     proj.extra_file_to_sign File.join(proj.bindir, 'pxp-agent')
     proj.extra_file_to_sign File.join(proj.bindir, 'wrapper.sh')
-    proj.signing_hostname 'osx-signer-prod-3.delivery.puppetlabs.net'
-    proj.signing_username 'jenkins'
-    proj.signing_command 'security -q unlock-keychain -p \$$OSX_SIGNING_KEYCHAIN_PW \$$OSX_SIGNING_KEYCHAIN; codesign --timestamp --keychain \$$OSX_SIGNING_KEYCHAIN -vfs \"\$$OSX_CODESIGNING_CERT\"'
+    proj.use_local_signing true
+    proj.signing_command 'codesign --timestamp --keychain $$OSX_SIGNING_KEYCHAIN -vfs "$$OSX_CODESIGNING_CERT"'
```

Comment:
Because I have it set up to run as root and imported into the System keychain on my VM, we don't need to specifically unlock the keychain. Later on, when we move this into GHA, we may want a separate keychain, or we may end up doing this entirely differently.